SimGeneral/DataMixingModule/plugins/DataMixingModule.cc

```diff
@@ -71,9 +72,9 @@ namespace edm
 
     // Check to see if we are working in Full or Fast Simulation
 
-    MergeTrackerDigis_ = (ps.getParameter<std::string>("TrackerMergeType")).compare("Digis") == 0;
-    MergeEMDigis_ = (ps.getParameter<std::string>("EcalMergeType")).compare("Digis") == 0;
-    MergeHcalDigis_ = (ps.getParameter<std::string>("HcalMergeType")).compare("Digis") == 0;
+    MergeTrackerDigis_ = (ps.getParameter<std::string>("TrackerMergeType")) == "Digis";
+    MergeEMDigis_ = (ps.getParameter<std::string>("EcalMergeType")) == "Digis";
+    MergeHcalDigis_ = (ps.getParameter<std::string>("HcalMergeType")) == "Digis";
```

Comment:
[just in case] these are apparently fictitious diffs: the changes were made already in the release with https://github.com/cms-sw/cmssw/commit/a922a87189ae1b83ffb5dabfc0ad6a2a62a8dca2